.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
 ###### Description
 
 
+<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->
```

Comment:
As long as the PR body contains `[skip notification]` it would work (even in a comment).